turbine/benches/retransmit_stage.rs

```diff
@@ -120,14 +120,14 @@ fn bench_retransmitter(bencher: &mut Bencher) {
 
     let num_packets = data_shreds.len();
 
-    let retransmitter_handles = retransmitter(
-        Arc::new(sockets),
-        quic_endpoint_sender,
+    let retransmit_stage = RetransmitStage::new(
         bank_forks,
         leader_schedule_cache,
         cluster_info,
+        Arc::new(sockets),
+        quic_endpoint_sender,
         shreds_receiver,
-        Arc::default(), // solana_rpc::max_slots::MaxSlots
```

Comment:
The comment here was to hint to the reader what type is initialized here.
Why remove it?

Reply:
Replaced with explicit instantiation (so it is compiler-enforced to be correct)